Makefile

```diff
@@ -1,7 +1,7 @@
-PREFIX ?=	$PREFIX
+include .config
```

Comment:
Sorry, caught up with other things. I was using OpenBSD; selfishly, I'm mostly interesting in ensuring this works on platforms I use (but not opposed to making it compatible widely). The configuration and Makefile set up also derives from a skeleton I was using a while ago to get projects set up quickly, so I'm not particularly tied to this approach if a better one can be found.